app/pages/Accounts/useAccountSync.ts

```diff
@@ -124,10 +129,11 @@ export default function useAccountSync(onError: (message: string) => void) {
                 force: true,
             })
         );
+        load.then(() => setIsLoadDone(true)).catch(noOp);
```

Comment:
Maybe this should actually be a finally? (though that might be a moot point, but I'm not sure when this could throw)

Reply:
Instead I have caught errors and use that to display the error.